src/packages/solid/.storybook/preview.tsx

```diff
@@ -55,6 +53,11 @@ const preview = {
   },
   decorators: [
     (Story) => {
+      useFocusManager({
+        Menu: 'm',
```

Comment:
you don't need to pass any key mapping it - it does defaults for left, right up down enter...